apps/sensenet/src/components/drawer/PermanentDrawer.tsx

```diff
@@ -129,6 +129,7 @@ export const PermanentDrawer = () => {
               PATHS.setup.appPath,
               PATHS.contentTypes.appPath,
               PATHS.localization.appPath,
+              PATHS.root.appPath,
```

Comment:
PATHS.custom.appPath.replace(':path', 'root'),
and root can be deleted from PATHS